### ISSUE TITLE
how can I work it with no ground_truth image?

### ISSUE THREAD

**Issue by DeepSpace98** [rvorias/ind_knn_ad#24]
Now I want to use patchcore to train,but my dateset don't include the ground_truth image.What should I do to let it work?

**Comment by rvorias** [comment]
You can still train on your dataset However, the current evaluation step will likely fail.

You have three options:
1) Customize `run.py` and don't call the built-in `evaluate`.

2) Change the `evaluate` function such that it doesn't expect a tuple of three elements at the dataloader. https://github.com/rvorias/ind_knn_ad/blob/57de628b334f9dd62316ac7770db6e963b71ee8e/indad/models.py#L64

3) Add dummy mask images.
